apache/superset
# style: Database Modal UI Polish

### SUMMARY
Corrected polishing tasks designated in [this Clubhouse ticket](https://app.clubhouse.io/preset/story/14661/ui-polish-bug-fixes), then matched the styling of the component as closely to the [Figma specifications](https://www.figma.com/file/NbuTOMaUdv2GCBoDyihdLw/Database-Connection?node-id=1583%3A2216) as possible

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<img width="502" alt="dbConnect-1" src="https://user-images.githubusercontent.com/55605634/122953901-98385e80-d344-11eb-876b-951750a4509d.png">
<img width="502" alt="dbConnect-basic" src="https://user-images.githubusercontent.com/55605634/122953911-9b334f00-d344-11eb-8f2e-6731380b7283.png">
<img width="502" alt="dbConnect-advancedSticky" src="https://user-images.githubusercontent.com/55605634/122953919-9cfd1280-d344-11eb-83bb-9a593aa7573d.png">
<img width="502" alt="dbEdit" src="https://user-images.githubusercontent.com/55605634/122953928-9e2e3f80-d344-11eb-9dcb-3e2dd94d0859.png">


### TESTING INSTRUCTIONS
From the Superset home page:
- Hover over "Data" in the upper navigation bar
- Click "Database"
- Click the "+ Database" button below the navigation bar and to the right
	- There are also changes in the Edit database modal
- Navigation through the database modal and observe the styling

### ADDITIONAL INFORMATION
